### DISCUSSION TITLE
Phone number/SSN Number/User name for login

### DISCUSSION THREAD

**Discussion by awnish-npg** [directus/directus#7205] [Feature Requests]
I need to implement the login either via Phone Number or SSN number or Username in my application. Please suggest of there is any workaround for this in the recent releases?

**Comment by rijkvanzanten** [comment]
See https://github.com/directus/directus/discussions/6670, https://github.com/directus/directus/discussions/4549, https://github.com/directus/directus/discussions/4237

**Comment by anlexN** [comment]
good idea
when does directus support registering and logging in with these methods?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
